src/main/java/com/dataloom/client/LoomClient.java

```diff
@@ -30,14 +30,14 @@
     public LoomClient( Environment environment, SerializableSupplier<String> jwtToken ) {
         this( () -> {
             final Retrofit retrofit = RetrofitFactory.newClient( environment, jwtToken );
-            return (ApiFactory) retrofit::create;
+            return (ApiFactory) clazz -> retrofit.create( clazz );
```

Comment:
this is interesting; would like to know more about this. I think Katherine and I both have compile errors locally with the `retrofit::create` version